.mergify.yml

```
@@ -122,7 +124,9 @@ pull_request_rules:
       - base=release-v2.0
       - label!=DNM
       - "#changes-requested-reviews-by=0"
-      - "#approved-reviews-by>=1"
+      - "#approved-reviews-by>=2"
+      - "approved-reviews-by=@ceph/ceph-csi-contributors"
+      - "approved-reviews-by=@ceph/ceph-csi-maintainers"
```

```suggestion
      - "#approved-reviews-by>=2"
      - "approved-reviews-by=@ceph/ceph-csi-contributors"
      - "approved-reviews-by=@ceph/ceph-csi-maintainers"
```

Backport prs also need two approvals now ?

---

Yes as we have more people let's have 2 approvals